privacy/ccpa/policy.go

```diff
@@ -14,21 +16,30 @@ type Policy struct {
 }
 
 // ReadFromRequestWrapper extracts the CCPA regulatory information from an OpenRTB bid request.
-func ReadFromRequestWrapper(req *openrtb_ext.RequestWrapper) (Policy, error) {
+func ReadFromRequestWrapper(req *openrtb_ext.RequestWrapper, gpp gpplib.GppContainer) (Policy, error) {
 	var consent string
 	var noSaleBidders []string
 
-	if req == nil {
-		return Policy{}, nil
+	for i, id := range gpp.SectionTypes {
+		if id == gppConstants.SectionUSPV1 {
+			//if id == 6 {
```

Comment:
Left over debugging comment?

Reply:
Left over from when `SectionUSPV1` was missing from the constants. So not exactly debugging, but needs to go.